### PR TITLE
2015 07 releases

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: beamlines/dev_env/configuration/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+build:
+  number: 1
+
+requirements:
+  run:
+    - conda_etc
+
+about:
+  summary: 'Development Enviroment server configuration'
```
Comment (ericdill) [inline beamlines/dev_env/configuration/meta.yaml]: Enviroment -> Environment

Comment (ericdill) [inline beamlines/dev_env/configuration/meta.yaml]: "Development environment configurations to point metadatastore and filestore at localhost"

File: beamlines/dev_env/configuration/build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+mkdir -p $PREFIX/etc
+ETC=$PREFIX/etc
+
+
+# set up
+echo "# metadatastore configuration
+# installed by nslsii_dev_configuration
+# DO NOT EDIT
+host: localhost
+database: datastore_DEV
```
Comment (ericdill) [inline beamlines/dev_env/configuration/build.sh]: can you change this to `-dev`

File: beamlines/dev_env/configuration/build.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+port: 27017
+timezone: US/Eastern
+" > $ETC/metadatastore.yml
+
+echo "# filestore configuration
+# installed by nslsii_dev_configuration
+# DO NOT EDIT
+host: localhost
+database: filestore_DEV
```
Comment (ericdill) [inline beamlines/dev_env/configuration/build.sh]: can you change this to `-dev`

**Comment by tacaswell** [comment]
@ericdill Done

**Comment by danielballan** [comment]
:carrot: <-- aspirational emoji

**Comment by ericdill** [comment]
download

- [ ] #34 
- [ ] NSLS-II/filestore#43
- [ ] NSLS-II/metadatastore#131

**Comment by ericdill** [comment]
You get the carrot!

**Pull request merged**